src/openapi.json

```diff
@@ -127,6 +127,30 @@
 				},
 				"required": ["id", "fileName", "sourceKey", "createdAt"]
 			},
+			"Organization": {
+				"type": "object",
+				"properties": {
+					"id": {
+						"type": "integer",
+						"readOnly": true,
+						"example": 3407
+					},
+					"employerIdentificationNumber": {
+						"type": "string",
+						"example": "111-11-1111"
+					},
+					"name": {
+						"type": "string",
+						"example": "Example Inc."
+					},
+					"createdAt": {
+						"type": "string",
+						"format": "date-time",
+						"readOnly": true
+					}
+				},
+				"required": ["id", "externalId", "optedIn", "createdAt"]
```

Comment:
I think this list is inaccurate? Probably copypasta from Applicant!

Reply:
Phewf thanks -- a little disappointed the linter didn't catch that!